src/taco-dependency-installer/installers/androidSdkInstaller.ts

```diff
@@ -37,7 +38,7 @@ class AndroidSdkInstaller extends InstallerBase {
     private static AndroidHomeName: string = "ANDROID_HOME";
     private static AndroidCommand = os.platform() === "win32" ? "android.bat" : "android";
     private static AndroidPackages: string[] = [
-        "tools",
+        // "tools",  // Android SDK comes by default with the tools package, so there is no need to update it. In the future, if we feel we want dependency installer to always install the latest Tools package, then uncomment this.
```

Comment:
Maybe add a comment noting that this was causing issues? Is there a known issue in the android installer we can point to?